dev-python/autobahn/autobahn-0.16.0.ebuild

```diff
@@ -0,0 +1,58 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
```

Comment:
TBH, I never thought about it, I was just using the template which was set for 5